cpp/rrtmgp_conversion.h

```diff
@@ -1172,6 +1197,66 @@ class SimpleNetCDF {
   }
 };
 
+// Copied with minor modes from YAKL_random.h
```

Comment:
If kokkos has a fast device-compatible pseudo-random-number generator library we could pivot to that too in the future.

Reply:
Yeah, the only issue is that the kokkos one gives different results so I was having trouble getting exact matches in the results.

Reply:
Yup, makes sense, I figured that would be the case.